feat: [PL-38838]: secret reference path test button (#1936)
harness/developer-hub: docs/platform/Secrets/2-add-use-text-secrets.md

* Azure Key Vault
* Hashicorp Vault
* GCP Secrets Manager

> Optionally you can test your secret refernce path validity using **Test** button

Reviewer: ```suggestion
You can validate the secret reference path using the **Test** button as shown below:
```
